lib/coxir/model/entities/message.ex

```diff
@@ -24,6 +24,7 @@ defmodule Coxir.Message do
     embeds_many(:embeds, Embed)
 
     embeds_one(:message_reference, Reference)
+    embeds_one(:components, Components)
```

Comment:
Judging from Discord's documentation, it seems that `components` is a list of objects.

The example [here](https://discord.com/developers/docs/interactions/message-components#what-are-components) shows the following structure of a message object:
```json

{
    "content": "This is a message with components",
    "components": [
        {
            "type": 1,
            "components": []
        }
    ]
}
```

But on the [message object](https://discord.com/developers/docs/resources/channel#message-object) they have `components` as an object and not a list. Looks like that's a typo!

So I suggest we change this to:
```suggestion
    embeds_many(:components, Components)
```